client/app/components/packages/landuse-form/edit.js

```diff
@@ -80,6 +85,8 @@ export default class LandUseFormComponent extends Component {
   removeRelatedAction(relatedAction, changeset) {
     removeFromHasMany(changeset, 'relatedActions', relatedAction);
 
+    this.recordsToDelete.push(relatedAction);
+
     relatedAction.deleteRecord();
   }
 }
```

Comment:
Wondering.... Do we also need to push removed BBL/taxlots to recordsToDelete?
Like within the "removeBbl" action within the ProjectGeography component?

![image](https://user-images.githubusercontent.com/3311663/92741610-a6783980-f333-11ea-8bdb-20e6142fa2c2.png)

Reply:
> Wondering.... Do we also need to push removed BBL/taxlots to recordsToDelete?
> Like within the "removeBbl" action within the ProjectGeography component?
> 

Good point! I just missed that, going to check on that/implement that now